src/aics-image-viewer/components/App/index.tsx

```diff
@@ -232,7 +232,7 @@ const App: React.FC<AppProps> = (props) => {
   };
 
   const setAllChannelsUnloaded = (numberOfChannels: number): void => {
-    setChannelVersions(new Array(numberOfChannels).fill(0));
+    setChannelVersions(new Array(numberOfChannels).fill(INITIAL_CHANNEL_VERSION));
   };
 
   const setOneChannelLoaded = (index: number): void => {
```

Comment:
Channel versions are incremented once they're loaded once!

Reply:
Looks like channel transfer functions are now getting recomputed rather than remapped on initial load. This is why I suggested `-1` for `INITIAL_CHANNEL_VERSION` initially. Though it does add some additional complexity: `setOneChannelLoaded` has to go from simply incrementing channel versions to something like `Math.max(version + 1, 1)`, and we need two different versions of `setAllChannelsUnloaded` for when channels are unloaded due to an image switch vs. due to loading new data from the same image, where the second does something like `setChannelVersions(getChannelVersions.map((version) => Math.min(version, 0)));`.

Not sure whether the additional complexity is worth the unification; I'm leaning towards yes, but I'm on the fence. If so, the representation of `channelVersions` should definitely get a few lines of documentation. (And honestly, it should already have documentation - the current representation just confused me, and I wrote it. The rationale for representing loaded state as a number rather than a boolean is described all the way back in #156.)

Reply:
Moved this to a new branch because of some complexity that was worth checking a little more thoroughly! It's now on `refactor/remove-channel-loaded-ref`.

Reply:
I'm all for any change that both makes things work correctly and improves the readability, maintainability, and simplicity of the codebase.  Do you really think we can get rid of the channelversions?